[v0.37.0] Updates for scenario syntax (#149)
redwoodjs/learn.redwoodjs.com: docs/tutorial2/adding-comments-to-the-schema.md

* **comment**: the name of the model this data is for
  * **one, two**: a friendly name given to the scenario data which you can reference in your tests
    * **data**: contains the actual data that will be put in the database
      * **name, message, post**: fields that correspond to the schema. In this case a **Comment** requires that it be related to a **Post**, so the scenario has a `post` key and values as well (using Prisma's [nested create syntax](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#nested-writes))
    * **select, include**: optionally, to customize the object to `select` or `include` related fields [using Prisma's syntax](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#create-a-related-record)

> When you receive the `scenario` argument in your test, input just unwraps from the parent `data` key and you can follow the similar object nesting in order to reference the fields, like `scenario.comment.one.name`.

Reviewer: ```suggestion
When you receive the `scenario` argument in your test, the `data` key gets unwrapped so that you can reference fields like `scenario.comment.one.name`.
```

Author: Oo this is way clearer. What was even I doing. lol.
